src/main/resources/migrations/00043_create_table_oauth_states.sql

```diff
@@ -0,0 +1,15 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_table-oauth_states
+CREATE TABLE oauth_states (
+  id BIGSERIAL PRIMARY KEY,
+  gateway_account_external_id VARCHAR(26) NOT NULL,
+  state VARCHAR(26) NOT NULL,
```

Comment:
Is `26` chars enough for the state? Could you give an example of the values? Thanks

Reply:
The state is some value that we generate and I gave it 26 chars so we can reuse the external id generator to populate it :)